code/src/java/plugin/encounter/EncounterModel.java

```diff
@@ -62,15 +50,9 @@ public EncounterModel(String parentDir)
 	 */
 	public int getCR()
 	{
-		ReadXML xml;
-		File f = new File(dir + File.separator + "4_1.xml"); //$NON-NLS-1$
-		int i;
 		float cr = 0;
 
-		xml = new ReadXML(f);
-		xml.getTable();
```

Comment:
Do we know this (and the constructor above it) have no side effects?  The older code is not always known for being properly object oriented